src/utils/data-manager.js

```diff
@@ -78,10 +78,23 @@ export default class DataManager {
       if (tableData.checked) {
         this.selectedCount++;
       }
-      return {
+      const newRow = {
         ...row,
         tableData
       };
+      if (
+        this.lastDetailPanelRow &&
+        this.lastDetailPanelRow.tableData === prevTableData
```

Comment:
We can use `this.lastDetailPanelRow?.tableData` to check if `lastDetailPanelRow` exist?

ref. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Would that require users to use Node 14? If that's not a problem then that sounds great.  Also if you guys would rather use some kind of ID in this comparison, like `id' and `uuid`, rather than an object ref comparison then I'd be up to test that out.